packages/frontend/src/features/metricsCatalog/components/visualization/MetricsVisualization.tsx

```diff
@@ -276,39 +362,12 @@ const MetricsVisualization: FC<Props> = ({
                                 style={{ userSelect: 'none' }}
                             />
 
-                            <RechartsTooltip
-                                {...(comparison.type ===
-                                    MetricExplorerComparison.NONE && {
-                                    formatter: (value) => [
-                                        value,
-                                        results?.metric.label,
-                                    ],
-                                })}
-                                labelFormatter={(label) =>
-                                    dayjs(label).format('MMM D, YYYY')
-                                }
-                                labelStyle={{
-                                    fontWeight: 500,
-                                    color: colors.gray[7],
-                                    fontSize: 13,
-                                }}
-                                contentStyle={{
-                                    fontSize: fontSizes.xs,
-                                    fontWeight: 500,
-                                    backgroundColor: colors.offWhite[0],
-                                    borderRadius: radius.md,
-                                    border: `1px solid ${colors.gray[2]}`,
-                                    boxShadow:
-                                        '0px 8px 8px 0px rgba(0, 0, 0, 0.08), 0px 0px 1px 0px rgba(0, 0, 0, 0.25)',
-                                    padding: spacing.sm,
-                                }}
-                            />
+                            <RechartsTooltip content={<CustomTooltip />} />
 
                             <Line
-                                name={legendConfig?.[0]}
+                                name="metric"
                                 type="linear"
-                                dataKey="metric"
-                                label={legendConfig?.[0]}
+                                dataKey="metric.value"
```

Comment:
before, `metric` was a property with a value - the value of the metric point

now, it has 2 keys: 'value' and 'label'